xml/ha_qdevice-qnetd.xml

```diff
@@ -0,0 +1,508 @@
+                           to the node.
+                           A letter <literal>V</literal> means that both nodes can communicate
+                           with each other. In a split-brain situation, one node would be
+                           set to <literal>V</literal> and the other node would be set to
+                           <literal>NV</literal>.
+                        </para>
+                     </listitem>
+                  </varlistentry>
+                  <varlistentry>
+                     <term><literal>MW</literal> (Master wins) or
+                           <literal>NMW</literal>(not master wins)</term>
+                     <listitem>
+                        <para>
+                           Shows if the quorum device <parameter>master_wins</parameter>
+                           flag is set. By default you see <literal>NMW</literal> (not master
+                           wins) as the flag is not set.
+                           See the man page votequorum_qdevice_master_wins(3) for more
+                           information.
+                        </para>
+                     </listitem>
+                  </varlistentry>
+               </variablelist>
+            </callout>
+         </calloutlist>
+      </example>
+
+      <para>
+         You can get a similar output if you want to observe the &qnet; server.
+         See <xref linkend="ex-ha-qdevice-crm-corosync-status-qnetd"/> for a detailed
+         look.
+      </para>
+      <example xml:id="ex-ha-qdevice-crm-corosync-status-qnetd">
+         <title>Output Status of &qnet; Server</title>
+         <screen>&prompt.root;<command>crm corosync status qnetd</command> <co xml:id="co-quorum-qnet-cmd"/>
+Cluster "hacluster": <co xml:id="co-quorum-qnet-name"/>
+    Algorithm:          Fifty-Fifty split <co xml:id="co-quorum-qnet-algo"/>
+    Tie-breaker:        Node with lowest node ID
+    Node ID 3232235777: <co xml:id="co-quorum-qnet-alice"/>
+        Client address:         ::ffff:&subnetI;.1:54732
+        HB interval:            8000ms
+        Configured node list:   3232235777, 3232235778
+        Ring ID:                aa10ab0.8
+        Membership node list:   3232235777, 3232235778
+        Heuristics:             Undefined (membership: Undefined, regular: Undefined)
+        TLS active:             Yes (client certificate verified)
+        Vote:                   ACK (ACK)
+    Node ID 3232235778:
+        Client address:         ::ffff:&subnetI;.2:43016
+        HB interval:            8000ms
+        Configured node list:   3232235777, 3232235778
+        Ring ID:                aa10ab0.8
+        Membership node list:   3232235777, 3232235778
+        Heuristics:             Undefined (membership: Undefined, regular: Undefined)
+        TLS active:             Yes (client certificate verified)
+        Vote:                   No change (ACK)</screen>
+       <calloutlist>
+          <callout arearefs="co-quorum-qnet-cmd">
```

Comment:
As this shows the output of 'crm corosync status qnet' I would remove this entry from the example and rather state the alternative command in the introduction above (see my code proposal).

Reply:
Command is corrected now.